### DISCUSSION TITLE
The controls don't work

### DISCUSSION THREAD

**Discussion by Edwar2910** [LizardByte/community#275] [Sunshine] [answered]
I just installed Sunshine to use it on my PsVita with Moonlight, it's a project that I found out about recently but I'm getting interested, I managed to link it to my PsVita and when I entered I tried to play a game and it didn't work for me, I tried moving with the joystick and nothing, then I tried moving with the touch screen and I did move the mouse, but I'm not interested in controlling the computer that way if I especially want to play games, so I'm asking, does anyone have an idea? How to get it to fence with the controls on everything?

**Comment by Edwar2910** [comment]
Or if it´s a way to configure and create a way that u can play in PC whit the controllers as a controller, and i dont need it to play games from steam, i want it to games hacked

**Comment by ReenigneArcher** [comment, marked as answer]
https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/usage.html#windows
